README.md

```diff
@@ -786,6 +786,12 @@ For more, see [Pylint](https://pypi.org/project/pylint/2.15.7/) on PyPI.
 | PLR1722 | ConsiderUsingSysExit | Consider using `sys.exit()` | 🛠 |
 | PLW0120 | UselessElseOnLoop | Else clause on loop without a break statement, remove the else and de-indent all the code inside it |  |
 
+### flake8-import-conventions
+
+| Code | Name | Message | Fix |
+| ---- | ---- | ------- | --- |
+| IC001 | ImportAliasIsNotConventional | `...` should be imported as `...` |  |
```

Comment:
Do you think it'd be a huge mistake to use a different prefix here? I'm trying to move towards using three-letter prefixes for all plugins, both for consistency and to avoid collisions in the future. We're already deviating on the error code a bit (since the plugin has IC002 and such), so maybe it's fine? Like `ICO` or `ICN`?

Reply:
Though `ICOOO1` can be ambiguous with `ICOOO1` in some fonts